test/internet/test-dgram-multicast-multi-process.js

```diff
@@ -1,18 +1,109 @@
 'use strict';
-var common = require('../common'),
-    assert = require('assert'),
-    dgram = require('dgram'),
-    util = require('util'),
-    Buffer = require('buffer').Buffer,
-    fork = require('child_process').fork,
-    LOCAL_BROADCAST_HOST = '224.0.0.114',
-    TIMEOUT = common.platformTimeout(5000),
-    messages = [
-      new Buffer('First message to send'),
-      new Buffer('Second message to send'),
-      new Buffer('Third message to send'),
-      new Buffer('Fourth message to send')
-    ];
+const common = require('../common'),
+      assert = require('assert'),
```

Comment:
Since you're changing all of these lines, it would be awesome to split them into individual `const` declaration statements.